applications/serial_lte_modem/src/slm_defines.h

```diff
@@ -19,7 +19,7 @@
 
 /** The maximum allowed length of data send/receive through the SLM */
 #define SLM_MAX_PAYLOAD_SIZE 1024 /** max size of payload sent in command mode */
-#define SLM_MAX_MESSAGE_SIZE 2048 /** align with NRF_MODEM_TLS_MAX_MESSAGE_SIZE */
+#define SLM_MAX_MESSAGE_SIZE NRF_SOCKET_TLS_MAX_MESSAGE_SIZE
```

Comment:
Need include "nrf_modem_limits.h"

Reply:
`NRF_SOCKET_TLS_MAX_MESSAGE_SIZE` is defined in `nrf_socket.h` in [sdk-nrfxlib#910](https://github.com/nrfconnect/sdk-nrfxlib/pull/910/files#diff-a0270bc06f25b0fdd777ba45f70c85367a127dcc8fba675a6a8ca23841b6e5cfR149)

Reply:
Added `#include <nrf_socket.h>`